challenges/advanced-variadic-generics2/question.py

```diff
@@ -0,0 +1,45 @@
+"""
+TODO:
+
+Define an "tupled" function that accepts a function and returns a function with some arguments in the form of a tuple.
```

Comment:
The description is not so clear on what it expects user to do, and why 
I'd suggest give it an example or put it in a more practical scene